spec/controllers/users/piv_cac_authentication_setup_controller_spec.rb

```diff
@@ -161,9 +161,8 @@
         end
 
         it 'sends a recovery information changed event' do
-          # Receives twice because one is sent when signing up with a second factor
           expect(PushNotification::HttpPush).to receive(:deliver).
-            with(PushNotification::RecoveryInformationChangedEvent.new(user: user)).twice
```

Comment:
**Question:** So the double notification was a bug in general?

Reply:
it seemed like it was getting sent from the user account setup and adding phone as a second factor, but in reality it was getting sent on any login.